meshiphi/mesh_generation/environment_mesh.py

```diff
@@ -221,6 +738,33 @@ def get_max_cellbox_id(self):
                 max_id = int(cellbox.get_id())
         return max_id
 
+    def remove_cellbox(self, cellbox):
+        """
+            removes the given cellbox from the mesh
+
+            Args:
+                cellbox (AggregatedCellBox): the cellbox to be removed
+        """
+        self.agg_cellboxes.remove(cellbox)
+
+        self.neighbour_graph.remove_node_and_update_neighbours(cellbox.get_id())
+
+    def add_cellbox(self, cellbox):
+        """
+            adds the given cellbox to the mesh
+        """
+        self.agg_cellboxes.append(cellbox)
+
+        # TODO: update neighbour graph
```

Comment:
Does this still need to be done here? If not, remove the comment.

Reply:
Good catch, I've removed the comment :)